share/chruby/chruby.sh

```diff
@@ -2,7 +2,7 @@ CHRUBY_VERSION="0.3.9"
 RUBIES=()
 
 for dir in "$PREFIX/opt/rubies" "$HOME/.rubies"; do
-	[[ -d "$dir" && -n "$(ls -A "$dir")" ]] && RUBIES+=("$dir"/*)
```

Comment:
You can simple quote `ls`:

``` bash
"$("ls" -A "$dir")"
```

Bash expands only unquoted aliases: https://www.gnu.org/software/bash/manual/html_node/Aliases.html#Aliases